onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -1086,10 +1089,47 @@ def aten_bitwise_and(self: TInt, other: TInt) -> TInt:
 
 
 @torch_op("aten::bitwise_left_shift")
-def aten_bitwise_left_shift(self: TInt, other: TInt) -> TInt:
+def aten_bitwise_left_shift_int16(self: INT16, other: INT16) -> INT16:
```

Comment:
I forgot about this. When we call `op_schema.type_constraints`, do we get `tensor{int16}` from it?

Reply:
I think so!

Reply:
Is `other` expected to be a non-negative number (in spite of the typing)? Hope we don't need to handle negative shifts?

Reply:
That's a good point. Currently all tests pass so I assume so? I will add a comment

Reply:
Done